rules/windows/lateral_movement_scheduled_task_target.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/11/20"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies remote scheduled task creations on a target host. This could be indicative of adversary lateral movement.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Remote Scheduled Task Creation"
+note = "Decode the base64 encoded tasks actions registry value to investigate the task configured action."
+risk_score = 47
+rule_id = "954ee7c8-5437-49ae-b2d6-2960883898e9"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+/* Task Scheduler service incoming connection followed by TaskCache registry modification  */
+
+sequence by host.id, process.entity_id with maxspan = 1m
+   [network where process.name : "svchost.exe" and
+   network.direction == "incoming" and source.port > 49151 and destination.port > 49151 and
+   source.address != "127.0.0.1" and source.address != "::1"
+   ]
+   [registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Schedule\\TaskCache\\Tasks\\*\\Actions"]
```

Comment:
nice, I didn't know this did a registry write.
we could also look for file mods to system32\tasks. no need to do both, but I don't know which will result in a better detection

Reply:
the file one gives only file name (equiv to task name), the registry helps to investigate within same alert the task action (see note)